From 74c443f9474245c28edddc0847d89ea4c82ba0b2 Mon Sep 17 00:00:00 2001
From: Matt DeVillier <matt.devillier@puri.sm>
Date: Mon, 18 May 2020 14:02:27 -0500
Subject: [PATCH 1/3] soc/skylake/me.c: Print status regardless of device
 enable state

Checking the CSE device status before printing means it will skip
printing for devices with the ME disabled, leaving the user no easy
way to verify the ME is properly disabled. Remove the check.

Test: build/boot Librem 13v4, verify ME status printed as expected
on device with disabled/neutered ME.

Signed-off-by: Matt DeVillier <matt.devillier@puri.sm>
Change-Id: Iaa4f4a369d878a52136c3479027443ea4e731a36
---
 src/soc/intel/skylake/me.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/soc/intel/skylake/me.c b/src/soc/intel/skylake/me.c
index 89491f89c3..08aceb3f83 100644
--- a/src/soc/intel/skylake/me.c
+++ b/src/soc/intel/skylake/me.c
@@ -188,9 +188,6 @@ void intel_me_status(void)
 	union me_hfsts3 hfs3;
 	union me_hfsts6 hfs6;
 
-	if (!is_cse_enabled())
-		return;
-
 	hfs1.data = me_read_config32(PCI_ME_HFSTS1);
 	hfs2.data = me_read_config32(PCI_ME_HFSTS2);
 	hfs3.data = me_read_config32(PCI_ME_HFSTS3);
-- 
2.25.1

